libcnb-test/README.md

```diff
@@ -191,13 +191,15 @@ fn dynamic_fixture() {
 Building with multiple buildpacks, using [`BuildConfig::buildpacks`]:
 
 ```rust,no_run
+use libcnb_data::buildpack_id;
```

Comment:
(End users won't have libcnb_data in their direct dependencies list, but instead libcnb)

```suggestion
use libcnb::data::buildpack_id;
```

Reply:
fyi - to make this change work i had to add `libcnb` as a dev dependency to `libcnb-test/Cargo.toml`